resources/key.rb

```diff
@@ -126,11 +129,22 @@
 end
 
 action :import do
-  execute 'gpg2: import key' do
-    command "#{gpg_cmd} --import #{new_resource.key_file}"
-    user new_resource.user
-    group new_resource.group
-    not_if { key_exists(new_resource) }
+  Array(new_resource.key_fingerprint).each do |key|
```

Comment:
This would be nicer on a resource `coerce` imo, `coerce: proc { |p| Array(p) }`. Then it'd be covered everywhere.

Reply:
Yeah, that's a little nicer to read with the coerce. It's the difference between doing the to Array function in properties or the resource.